### PR TITLE
Demo instance

### PR DESCRIPTION
**Pull request by afwillia**

### CODE REVIEW

**Comment by afwillia** [comment]
1 - don't make any changes to the workflow file now. We won't need a separate demo-instance branch.
2 - remove HTAN image and references, but keep top menu and add a Sage image

File: ui.R [hunk 1]
```diff
@@ -12,43 +12,43 @@ ui <- shinydashboardPlus::dashboardPage(
...
-                inputId = paste0("header_dropdown_", x),
-                label = NULL,
-                choices = character(0)
-              )
-            )
-          }),
-          actionButton("btn_header_update", NULL, icon("sync-alt"), class = "btn-shiny-effect")
-        )
-      )
-    ),
```
Comment (rrchai) [inline ui.R:40]: suggest to keep this chunk.

**Comment by afwillia** [comment]
Not sure why server.R, dcWater.R, and .gitignore are showing in the diff - they look the same to me when inspecting the actual files.

**Comment by rrchai** [comment]
@afwillia Yeah, the pr is sometimes not updated with the latest changes in `develop`. A little trick of solving this issue is to change base branch to another branch and then switch back.

**Review by rrchai — APPROVED** [review]
LGTM!

**Pull request merged**